csrc/mmdeploy/apis/java/README.md

```diff
@@ -0,0 +1,25 @@
+# Build Java API
+
+## From Source
+
+### Requirements
+
+- OpenJDK >= 10
+
+### Installation
+
+For using java apis, you should build java class and c++ sdk.
+
+**Step 1.** Build java class.
+
+Build java `.class` files.
+
+```bash
+cd csrc/mmdeploy/apis/java
+javac mmdeploy/*.java
```

Comment:
`git status` shows a bunch of `*.class` files are untracked in diectory `csrc/mmdeploy/apis/java/mmdeploy/` after building. 
Please add them to `.gitignore`.